ql-qlf-plugin/Makefile

```diff
@@ -9,7 +9,8 @@
 NAME = ql-qlf
 SOURCES = synth_quicklogic.cc \
           ql-dsp.cc \
-          pp3_braminit.cc
+          pp3_braminit.cc \
+	  quicklogic_eqn.cc
```

Comment:
This seems to be misaligned with the rest

Reply:
Corrected it